### DISCUSSION TITLE
Datenpunkte beim Lesen und Setzen berechnen

### DISCUSSION THREAD

**Discussion by schluo** [Zefau/ioBroker.jarvis#2825] [Q&A]
Hallo Jarvis-Gemeinde,
wahrscheinlich nur eine Kleinigkeit:
Ich habe einen Datenpunkt eines Thermostats, der Temperaturen abspeichert, jedoch mit dem Faktor 5 --> 20 Grad = 100
Die Temperatur soll mit einem "LevelBody" gesteuert/eingestellt werden.
Beim Anzeigen ist das kein Problem: `{"value":"val => val / 5","min":15,"max":25}`
Beim Setzen des Wertes muss nun natürlich wieder umgekehrt gerechnet werden. Der gewählte Wert muss *5 im Datenpunkt gespeichert werden. Wie stelle ich das am besten an ohne im IOBroker die Werte umzurechnen?
Danke und schöne Grüße,
Oliver

**Comment by mcuiobroker** [comment]
Keine Umrechnung in den Datenpunkt-Eigenschaften.
Nur die Anzeige im LevelBody Optionen anpassen
![image](https://github.com/user-attachments/assets/694471f0-29fe-437b-a33c-0fccbbb401bb)
